packages/indy-vdr/src/__tests__/IndyVdrModule.test.ts

```diff
@@ -1,5 +1,7 @@
 import type { DependencyManager } from '@aries-framework/core'
 
+import { indyVdr } from '@hyperledger/indy-vdr-shared'
```

Comment:
Please see my other comment about which of the packages should be used where and carefully review this pr yourself to make sure we're importing from the right pacakges